contrib/debootstrap/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "debootstrap"
+pkgver = "1.0.137"
+pkgrel = 0
+build_style = "makefile"
+depends = [
+    "debian-archive-keyring",
+    "gnupg",
+    "llvm-binutils",
+    "mount",
+    "wget2",
+    "xz",
+    "zstd-progs",
```

Comment:
in general use cmd: virtual package names for whatever commands are needed by this